src/app/overview/page.tsx

```
@@ -1,8 +1,14 @@
+import AboutSection from "@components/features/about-section/about-section";
+import CompaniesSection from "@components/features/companies-section/companies-section";
+
+import GallerySection from "@components/features/gallary-section/gallery-section";
 
 export default function HomePage() {
     return (
-    <>
-        <p>hello world</p>
-    </>
+        <div className="mx-auto flex w-[1281px] flex-col gap-32 py-20">
```

ESLint

[error] className value must use cn() function. Example: className={cn("...")}

(local-rules/require-cn-classname)